eq-author-api/src/validation/schemas/entities.json

```diff
@@ -96,68 +89,102 @@
           "$ref": "entities.json#/definitions/totalValidation"
         }
       },
-      "if": {
-        "properties": {
-          "pageType": {
-            "enum": [
-              "QuestionPage"
-            ]
-          }
-        }
-      },
-      "then": {
-        "properties": {
-          "answers": {
-            "type": "array",
-            "items": {
-              "$ref": "entities.json#/definitions/answer"
-            },
-            "minItems": 1,
-            "errorMessage": {
-              "minItems": "ERR_NO_ANSWERS"
-            }
-          }
-        }
-      },
-      "else": {
-        "if": {
-          "properties": {
-            "pageType": {
-              "enum": [
-                "CalculatedSummaryPage"
-              ]
+      "allOf": [
```

Comment:
You've put the tests inside an allOf here, rather than testing each field in the properties. I would only do if this if testing multiple related senarios around the same field.